.github/workflows/kfam_oci_publish.yaml

```diff
@@ -0,0 +1,65 @@
+name: Build & Publish KFAM OCI image
+on:
+  push:
+    branches:
+      - main
+      - v*-branch
+    paths:
+      - components/access-management/**
+      - releasing/version/VERSION
+
```

Comment:
Sorry i missed to add .github/workflows/kfam_oci_publish.yaml as trigger path here. Please create a follow up PR @yehudit1987